[2021-12-28 13:36:51,696] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_sligo.persist_data scheduled__2021-12-27T10:45:00+00:00 [queued]>
[2021-12-28 13:36:51,831] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_sligo.persist_data scheduled__2021-12-27T10:45:00+00:00 [queued]>
[2021-12-28 13:36:51,832] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:36:51,836] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-28 13:36:51,841] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:36:51,974] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): persist_data> on 2021-12-27 10:45:00+00:00
[2021-12-28 13:36:52,070] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_sligo', 'persist_data', 'scheduled__2021-12-27T10:45:00+00:00', '--job-id', '5085', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_sligo.py', '--cfg-path', '/tmp/tmpjh3ems3d', '--error-file', '/tmp/tmpjxwlrovw']
[2021-12-28 13:36:52,021] {standard_task_runner.py:52} INFO - Started process 50978 to run task
[2021-12-28 13:36:52,078] {standard_task_runner.py:77} INFO - Job 5085: Subtask persist_data
[2021-12-28 13:36:52,366] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_sligo.persist_data scheduled__2021-12-27T10:45:00+00:00 [running]> on host 0c125a5bba95
[2021-12-28 13:36:52,827] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_sligo
AIRFLOW_CTX_TASK_ID=persist_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-27T10:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-27T10:45:00+00:00
[2021-12-28 13:36:52,958] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-28 13:36:52,975] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name persist data --verbose /opt/***/jobs/persist_db.py sligo
[2021-12-28 13:37:03,298] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-28 13:37:03,701] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-28 13:37:03,703] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-28 13:37:03,704] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-28 13:37:03,706] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-28 13:37:03,707] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-28 13:37:03,808] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-28 13:37:03,810] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-28 13:37:03,811] {spark_submit.py:514} INFO - deployMode              null
[2021-12-28 13:37:03,815] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-28 13:37:03,816] {spark_submit.py:514} INFO - executorCores           null
[2021-12-28 13:37:03,817] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-28 13:37:03,819] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-28 13:37:03,820] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-28 13:37:03,821] {spark_submit.py:514} INFO - driverCores             null
[2021-12-28 13:37:03,823] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-28 13:37:03,824] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-28 13:37:03,826] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-28 13:37:03,827] {spark_submit.py:514} INFO - supervise               false
[2021-12-28 13:37:03,828] {spark_submit.py:514} INFO - queue                   null
[2021-12-28 13:37:03,832] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-28 13:37:03,833] {spark_submit.py:514} INFO - files                   null
[2021-12-28 13:37:03,834] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-28 13:37:03,835] {spark_submit.py:514} INFO - archives                null
[2021-12-28 13:37:03,836] {spark_submit.py:514} INFO - mainClass               null
[2021-12-28 13:37:03,837] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/persist_db.py
[2021-12-28 13:37:03,838] {spark_submit.py:514} INFO - name                    persist data
[2021-12-28 13:37:03,839] {spark_submit.py:514} INFO - childArgs               [sligo]
[2021-12-28 13:37:03,840] {spark_submit.py:514} INFO - jars                    null
[2021-12-28 13:37:03,841] {spark_submit.py:514} INFO - packages                null
[2021-12-28 13:37:03,842] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-28 13:37:03,843] {spark_submit.py:514} INFO - repositories            null
[2021-12-28 13:37:03,844] {spark_submit.py:514} INFO - verbose                 true
[2021-12-28 13:37:03,846] {spark_submit.py:514} INFO - 
[2021-12-28 13:37:03,847] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-28 13:37:03,849] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-28 13:37:03,850] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:37:03,852] {spark_submit.py:514} INFO - 
[2021-12-28 13:37:03,855] {spark_submit.py:514} INFO - 
[2021-12-28 13:37:04,582] {spark_submit.py:514} INFO - Main class:
[2021-12-28 13:37:04,584] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-28 13:37:04,586] {spark_submit.py:514} INFO - Arguments:
[2021-12-28 13:37:04,589] {spark_submit.py:514} INFO - file:/opt/***/jobs/persist_db.py
[2021-12-28 13:37:04,593] {spark_submit.py:514} INFO - null
[2021-12-28 13:37:04,595] {spark_submit.py:514} INFO - sligo
[2021-12-28 13:37:04,597] {spark_submit.py:514} INFO - --verbose
[2021-12-28 13:37:04,599] {spark_submit.py:514} INFO - Spark config:
[2021-12-28 13:37:04,601] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:37:04,603] {spark_submit.py:514} INFO - (spark.app.name,persist data)
[2021-12-28 13:37:04,605] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-28 13:37:04,609] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-28 13:37:04,611] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-28 13:37:04,621] {spark_submit.py:514} INFO - 
[2021-12-28 13:37:04,624] {spark_submit.py:514} INFO - 
[2021-12-28 13:37:04,626] {spark_submit.py:514} INFO - 
[2021-12-28 13:37:06,906] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-28 13:37:06,931] {spark_submit.py:514} INFO - 21/12/28 13:37:06 INFO SparkContext: Running Spark version 3.2.0
[2021-12-28 13:37:07,185] {spark_submit.py:514} INFO - 21/12/28 13:37:07 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-28 13:37:07,688] {spark_submit.py:514} INFO - 21/12/28 13:37:07 INFO ResourceUtils: ==============================================================
[2021-12-28 13:37:07,694] {spark_submit.py:514} INFO - 21/12/28 13:37:07 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-28 13:37:07,703] {spark_submit.py:514} INFO - 21/12/28 13:37:07 INFO ResourceUtils: ==============================================================
[2021-12-28 13:37:07,708] {spark_submit.py:514} INFO - 21/12/28 13:37:07 INFO SparkContext: Submitted application: Persist DB
[2021-12-28 13:37:07,878] {spark_submit.py:514} INFO - 21/12/28 13:37:07 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-28 13:37:07,947] {spark_submit.py:514} INFO - 21/12/28 13:37:07 INFO ResourceProfile: Limiting resource is cpu
[2021-12-28 13:37:07,957] {spark_submit.py:514} INFO - 21/12/28 13:37:07 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-28 13:37:08,247] {spark_submit.py:514} INFO - 21/12/28 13:37:08 INFO SecurityManager: Changing view acls to: ***
[2021-12-28 13:37:08,248] {spark_submit.py:514} INFO - 21/12/28 13:37:08 INFO SecurityManager: Changing modify acls to: ***
[2021-12-28 13:37:08,250] {spark_submit.py:514} INFO - 21/12/28 13:37:08 INFO SecurityManager: Changing view acls groups to:
[2021-12-28 13:37:08,251] {spark_submit.py:514} INFO - 21/12/28 13:37:08 INFO SecurityManager: Changing modify acls groups to:
[2021-12-28 13:37:08,252] {spark_submit.py:514} INFO - 21/12/28 13:37:08 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-28 13:37:09,600] {spark_submit.py:514} INFO - 21/12/28 13:37:09 INFO Utils: Successfully started service 'sparkDriver' on port 40533.
[2021-12-28 13:37:09,770] {spark_submit.py:514} INFO - 21/12/28 13:37:09 INFO SparkEnv: Registering MapOutputTracker
[2021-12-28 13:37:09,889] {spark_submit.py:514} INFO - 21/12/28 13:37:09 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-28 13:37:09,953] {spark_submit.py:514} INFO - 21/12/28 13:37:09 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-28 13:37:09,958] {spark_submit.py:514} INFO - 21/12/28 13:37:09 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-28 13:37:09,976] {spark_submit.py:514} INFO - 21/12/28 13:37:09 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-28 13:37:10,040] {spark_submit.py:514} INFO - 21/12/28 13:37:10 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-2854c26a-d6b9-4b00-8fbd-186fd7f90fd5
[2021-12-28 13:37:10,089] {spark_submit.py:514} INFO - 21/12/28 13:37:10 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-28 13:37:10,126] {spark_submit.py:514} INFO - 21/12/28 13:37:10 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-28 13:37:10,791] {spark_submit.py:514} INFO - 21/12/28 13:37:10 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-28 13:37:10,851] {spark_submit.py:514} INFO - 21/12/28 13:37:10 INFO Utils: Successfully started service 'SparkUI' on port 4041.
[2021-12-28 13:37:11,076] {spark_submit.py:514} INFO - 21/12/28 13:37:11 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://0c125a5bba95:4041
[2021-12-28 13:37:12,314] {spark_submit.py:514} INFO - 21/12/28 13:37:12 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:37:12,550] {spark_submit.py:514} INFO - 21/12/28 13:37:12 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:37:12,551] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:37:12,552] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:37:12,554] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:37:12,555] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:37:12,556] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:37:12,557] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:37:12,559] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:37:12,560] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:37:12,561] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:37:12,562] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:37:12,567] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:37:12,569] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:37:12,572] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:37:12,573] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:37:12,574] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:37:12,575] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:37:12,576] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:37:12,577] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:37:12,578] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:37:12,579] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:37:12,580] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:37:12,585] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:37:12,588] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:37:12,590] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:37:12,591] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:37:12,593] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:37:12,594] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:37:12,596] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:37:12,597] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:37:12,599] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:37:12,600] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:37:12,602] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:37:12,603] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:37:12,605] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:37:12,606] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:37:12,608] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:37:12,609] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:37:12,615] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:37:12,616] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:37:12,621] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:37:12,623] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:37:12,625] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:37:12,626] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:37:12,630] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:37:12,632] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:37:12,633] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:37:12,634] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:37:12,636] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:37:12,637] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:37:12,638] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:37:12,640] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:37:12,641] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:37:12,644] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:37:12,645] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:37:12,646] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:37:12,647] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:37:12,648] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:37:32,313] {spark_submit.py:514} INFO - 21/12/28 13:37:32 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:37:32,336] {spark_submit.py:514} INFO - 21/12/28 13:37:32 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:37:32,337] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:37:32,338] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:37:32,339] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:37:32,340] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:37:32,341] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:37:32,343] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:37:32,344] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:37:32,345] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:37:32,346] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:37:32,348] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:37:32,350] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:37:32,352] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:37:32,353] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:37:32,354] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:37:32,355] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:37:32,356] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:37:32,357] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:37:32,358] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:37:32,359] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:37:32,360] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:37:32,361] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:37:32,362] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:37:32,363] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:37:32,364] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:37:32,367] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:37:32,369] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:37:32,370] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:37:32,371] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:37:32,372] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:37:32,374] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:37:32,375] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:37:32,376] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:37:32,377] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:37:32,378] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:37:32,380] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:37:32,382] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:37:32,385] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:37:32,386] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:37:32,389] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:37:32,394] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:37:32,401] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:37:32,403] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:37:32,405] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:37:32,406] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:37:32,407] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:37:32,409] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:37:32,410] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:37:32,412] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:37:32,414] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:37:32,415] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:37:32,419] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:37:32,420] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:37:32,421] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:37:32,423] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:37:32,424] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:37:32,425] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:37:32,427] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:37:52,280] {spark_submit.py:514} INFO - 21/12/28 13:37:52 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:37:52,300] {spark_submit.py:514} INFO - 21/12/28 13:37:52 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:37:52,301] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:37:52,302] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:37:52,303] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:37:52,304] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:37:52,305] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:37:52,305] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:37:52,306] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:37:52,307] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:37:52,308] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:37:52,308] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:37:52,309] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:37:52,311] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:37:52,311] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:37:52,312] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:37:52,313] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:37:52,313] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:37:52,314] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:37:52,315] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:37:52,315] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:37:52,316] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:37:52,317] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:37:52,318] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:37:52,318] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:37:52,319] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:37:52,320] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:37:52,321] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:37:52,322] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:37:52,322] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:37:52,323] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:37:52,324] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:37:52,325] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:37:52,325] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:37:52,326] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:37:52,327] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:37:52,328] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:37:52,328] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:37:52,329] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:37:52,330] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:37:52,330] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:37:52,331] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:37:52,332] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:37:52,332] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:37:52,333] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:37:52,334] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:37:52,334] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:37:52,335] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:37:52,336] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:37:52,336] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:37:52,337] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:37:52,337] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:37:52,338] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:37:52,339] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:37:52,339] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:37:52,340] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:37:52,340] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:37:52,341] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:37:52,341] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:38:12,245] {spark_submit.py:514} INFO - 21/12/28 13:38:12 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-28 13:38:12,246] {spark_submit.py:514} INFO - 21/12/28 13:38:12 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-28 13:38:12,258] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 33555.
[2021-12-28 13:38:12,259] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO NettyBlockTransferService: Server created on 0c125a5bba95:33555
[2021-12-28 13:38:12,260] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-28 13:38:12,274] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO SparkUI: Stopped Spark web UI at http://0c125a5bba95:4041
[2021-12-28 13:38:12,285] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-28 13:38:12,286] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 0c125a5bba95, 33555, None)
[2021-12-28 13:38:12,291] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-28 13:38:12,294] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO BlockManagerMasterEndpoint: Registering block manager 0c125a5bba95:33555 with 434.4 MiB RAM, BlockManagerId(driver, 0c125a5bba95, 33555, None)
[2021-12-28 13:38:12,300] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 0c125a5bba95, 33555, None)
[2021-12-28 13:38:12,301] {spark_submit.py:514} INFO - 21/12/28 13:38:12 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-28 13:38:12,303] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 0c125a5bba95, 33555, None)
[2021-12-28 13:38:12,322] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-28 13:38:12,359] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO MemoryStore: MemoryStore cleared
[2021-12-28 13:38:12,361] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO BlockManager: BlockManager stopped
[2021-12-28 13:38:12,375] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-28 13:38:12,380] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-28 13:38:12,401] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO SparkContext: Successfully stopped SparkContext
[2021-12-28 13:38:12,551] {spark_submit.py:514} INFO - 21/12/28 13:38:12 ERROR SparkContext: Error initializing SparkContext.
[2021-12-28 13:38:12,552] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:38:12,552] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:38:12,553] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:38:12,554] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:38:12,555] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:38:12,556] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:38:12,557] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:38:12,558] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:38:12,559] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:38:12,560] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:38:12,561] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:38:12,563] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:38:12,564] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:38:12,564] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:38:12,565] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:38:12,566] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:38:12,567] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:38:12,568] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO SparkContext: SparkContext already stopped.
[2021-12-28 13:38:12,569] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-28 13:38:12,571] {spark_submit.py:514} INFO - File "/opt/***/jobs/persist_db.py", line 52, in <module>
[2021-12-28 13:38:12,572] {spark_submit.py:514} INFO - .config("spark.driver.bindAddress", "localhost") \
[2021-12-28 13:38:12,573] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-28 13:38:12,574] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-28 13:38:12,575] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-28 13:38:12,577] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-28 13:38:12,578] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-28 13:38:12,579] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-28 13:38:12,580] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-28 13:38:12,581] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-28 13:38:12,582] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:38:12,583] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:38:12,584] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:38:12,585] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:38:12,586] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:38:12,587] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:38:12,588] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:38:12,589] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:38:12,590] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:38:12,591] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:38:12,592] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:38:12,593] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:38:12,594] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:38:12,595] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:38:12,596] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:38:12,597] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:38:12,598] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:38:12,599] {spark_submit.py:514} INFO - 
[2021-12-28 13:38:12,688] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO ShutdownHookManager: Shutdown hook called
[2021-12-28 13:38:12,690] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO ShutdownHookManager: Deleting directory /tmp/spark-cf67f663-9537-4545-9fe0-97efc0ddf60a
[2021-12-28 13:38:12,699] {spark_submit.py:514} INFO - 21/12/28 13:38:12 INFO ShutdownHookManager: Deleting directory /tmp/spark-ba141a98-a42c-45a9-b0cb-27c1a112ed4b
[2021-12-28 13:38:12,854] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name persist data --verbose /opt/***/jobs/persist_db.py sligo. Error code is: 1.
[2021-12-28 13:38:12,873] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_sligo, task_id=persist_data, execution_date=20211227T104500, start_date=20211228T133651, end_date=20211228T133812
[2021-12-28 13:38:12,899] {standard_task_runner.py:91} ERROR - Failed to execute job 5085 for task persist_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name persist data --verbose /opt/***/jobs/persist_db.py sligo. Error code is: 1.
[2021-12-28 13:38:12,919] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-28 13:38:13,080] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
